apps/condo/domains/common/utils/codegeneration/generate.server.utils.js

```diff
@@ -1,21 +1,67 @@
-const { pickBy, get, isEmpty } = require('lodash')
+const { pickBy, get, isEmpty, isObject } = require('lodash')
 
 const conf = require('@core/config')
 const { getById } = require('@core/keystone/schema')
+const { GQLError } = require('@core/keystone/errors')
 
 const IS_DEBUG = conf.NODE_ENV === 'development' || conf.NODE_ENV === 'test'
 
 const isNotUndefined = (x) => typeof x !== 'undefined'
+const ALLOWED_OPTIONS = ['errorMapping', 'doesNotExistError', 'multipleObjectsError']
 
-function _throwIfError (context, errors, data, errorMessage) {
+function _getAllErrorMessages (errors) {
+    const messages = []
+    for (const x of errors) {
+        const m = get(x, 'message', '')
```

Comment:
There is no need to use default value here
```suggestion
        const m = get(x, 'message')
```